getting_started/eeglab.md

```diff
@@ -9,34 +9,66 @@ tags: [dataformat, eeglab, eeg]
+![How does FieldTrip use EEGLAB](/assets/img/getting_started/eeglab/FieldTrip_uses_EEGLAB.png)
+{% include image src="/assets/img/getting_started/eeglab/FieldTrip_uses_EEGLAB.png" width="300" %}
 
-To prevent overlapping programming efforts, the DIPFIT version 2.0 plugin uses the forward and inverse methods that are already implemented in FieldTrips **[ft_dipolefitting](https://github.com/fieldtrip/fieldtrip/blob/release/ft_dipolefitting.m)**) function. That means that you can use the DIPFIT graphical interface in EEGLAB, but you should have installed FieldTrip on your MATLAB path.
 
-## Using other FieldTrip functions
+## How does EEGLAB use FieldTrip?
 
-Besides using FieldTrip as the underlying "engine" for the DIPFIT plug-in of EEGLAB, it is also possible to export various aspects of your EEGLAB data structure completely to FieldTrip. Subsequently, you can continue your analysis of the data using the command-line FieldTrip functions
+EEGLAB integrates FieldTrip functions through the DIPFIT plug-in. This tool allows the localisation of the source signals from the components that have been separated using ICA. The main function used by DIPFIT is **[ft_dipolefitting](https://github.com/fieldtrip/fieldtrip/blob/release/ft_dipolefitting.m)**. It performs a grid search and non-linear fit with one or multiple dipoles and try to find the location where the dipole model is best able to explain the measured EEG topography, using realistic BEM and FEM volume conduction models.
+The following figure illustrates some FieldTrip functions used in DIPFIT:
 
-Exporting data from EEGLAB to FieldTrip most conveniently done on the MATLAB command line using the **eeglab2fieldtrip** function. That function takes an "EEG" MATLAB structure as input, and will give a MATLAB structure as output that is identical to the output format of one of the standard FieldTrip functions, for example **[ft_preprocessing](https://github.com/fieldtrip/fieldtrip/blob/release/ft_preprocessing.m)**, **[ft_timelockanalysis](https://github.com/fieldtrip/fieldtrip/blob/release/ft_timelockanalysis.m)** or **[ft_freqanalysis](https://github.com/fieldtrip/fieldtrip/blob/release/ft_freqanalysis.m)**.
+![How does EEGLAB use FieldTrip](/assets/img/getting_started/eeglab/EEGLAB_uses_FieldTrip.png)
+{% include image src="/assets/img/getting_started/eeglab/EEGLAB_uses_FieldTrip.png" width="300" %}
 
-The alternative to **eeglab2fieldtrip** is to save the data to disk and use the standard FieldTrip reading and preprocessing functions to read the data directly from disk.
 
-## Converting data between EEGLAB and FieldTrip
+Note: DIPFIT graphical interface can be used in EEGLAB, but FieldTrip should be installed on the MATLAB path.
```

Comment:
this sentence is still a bit weird. I think it should be removed.

Reply:
I decided to just modify it in a clearer way, finally.